### ISSUE TITLE
[Question] Can the Regexps be re-written to avoid backreferences?

### ISSUE THREAD

**Issue by anicholson** [orgapp/orgajs#23]
I'm trying out orga for a project that would let me embed the parser in the client-side. Unfortunately, the regexps in the parser use backreferences - something only Chrome supports at the moment.

Is it possible to re-write those in a more browser-compliant way?

:heart:

**Comment by xiaoxinghu** [comment]
I think by `backreferences` you mean lookbehind assertions, I have removed the only usage of it (I think that's the only place 😅) from the code base, and published a prerelease, please try it out and come back with issues if any. If all good then I will make a release.
Install it with `npm i orga@next`

**Comment by titaniumbones** [comment]
I'm not sure htis fixed the issue' in firefox I still get am "invalid regexp rexpression" error when trying to load orga using browserify.

**Comment by xiaoxinghu** [comment]
@titaniumbones yes, you are right, I see the same thing too in firefox. But it works fine in Chrome and Safari for now. Will have a look at the firefox issue later. I've also created a playground page here: [https://orga.js.org](https://orga.js.org) if you are interested.
